extensions/coroutines-extensions/src/commonMain/kotlin/com/squareup/sqldelight/runtime/coroutines/FlowExtensions.kt

```diff
@@ -33,22 +33,18 @@ import kotlin.jvm.JvmName
 import kotlin.jvm.JvmOverloads
 
 /** Turns this [Query] into a [Flow] which emits whenever the underlying result set changes. */
+@ExperimentalCoroutinesApi
```

Comment:
This is a bit concerning. I think I would rather suppress this instead of propagating it since we already propagate the flow preview annotation. If flow goes stable prior to the channel method, we'll have to switch back to the old version.

Reply:
Removed propagation, suppressed the experimental API usage warning and added a TODO.